code/game/objects/items/stacks/packagewrap.dm

```diff
@@ -140,9 +140,9 @@
 
 /obj/item/delivery/attack_self(mob/user as mob)
 	user.drop_item(src, user.loc)
-	if(contents.len)
-		if(ishuman(user))
-			user.put_in_hands(contents[1])
+	if(ishuman(user))
+		for(obj/item/M in contents)
```

Comment:
for(var/obj/item/M in contents)

(var)iable is important because you're doing something called "prototyping" and it's basically the bread and butter for this sort of stuff

Reply:
It's more than this code does not compile, because the `for` expects an iterator, and you're not giving it any.